ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -409,6 +412,32 @@ User.Response = Backbone.AssociatedModel.extend({
       }
     }
   },
+  canRead: function(perms) {
```

Comment:
❓ Does this still work if the Subject Identity Attribute is configured to something other than email?  The access values should match the configured attribute.

https://github.com/codice/ddf/blob/master/platform/security/core/security-core-services/src/main/java/ddf/security/impl/SubjectIdentityImpl.java
https://github.com/codice/ddf/blob/master/platform/security/core/security-core-services/src/main/resources/OSGI-INF/metatype/metatype.xml#L38

Reply:
Good point. I'll need to fix this